src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,25 @@
+package domain
+
+class Board(
+    val height: Int,
+    val width: Int,
+    val mineCount: Int,
+) {
+    private val board: MutableList<Space> = MutableList(height * width) { Space.Empty }
+
+    init {
+        plantMines()
+    }
+
+    fun board(): List<Space> {
+        return board
+    }
```

Comment:
```suggestion
    val board: List<Space> get() =  _board
```
좀더 코틀린스럽게 사용해볼순 없을까요?
https://kotlinlang.org/docs/properties.html#backing-properties

Reply:
board()함수는 List<Space>를 리턴하지만, 
board는 사실 MutableList네요, 형변환을 사용하면, 외부에서도 접근할수 있는구조가 될거같아요!
방어적 복사를 사용해보아요!